helm/pay-wallet-values-dev.yaml

```diff
@@ -84,6 +84,7 @@ microservice-chart:
     WALLET_EXPIRATION_QUEUE_NAME: "pagopa-d-itn-pay-wallet-expiration-queue"
     AZURE_EVENTHUB_TOPIC_NAME: "payment-wallet-ingestion-dl"
     AZURE_EVENTHUB_BOOTSTRAP_SERVER: "pagopa-d-itn-observ-evh.servicebus.windows.net:9093"
+    AZURE_EVENTHUB_CONNECTION_STRING: "kafka:29092"
```

Comment:
AZURE_EVENTHUB_CONNECTION_STRING it is an envSecret in main and thus conflicts with with this branch

Reply:
ops!

Reply:
fixed here d90cfc1d69c2572c34ff107ea143afe88dc9f1d4